client/battle/CBattleInterface.cpp

```diff
@@ -1416,9 +1416,11 @@ CBattleInterface::PossibleActions CBattleInterface::getCasterAction(const CSpell
 void CBattleInterface::setHeroAnimation(ui8 side, int phase)
 {
 	if(side == BattleSide::ATTACKER)
-		attackingHero->setPhase(phase);
+		if(attackingHero)
+			attackingHero->setPhase(phase);
 	else
-		defendingHero->setPhase(phase);
+		if(defendingHero)
+			defendingHero->setPhase(phase);

```

Comment:
I think we decide explicitly that something like that would need braces.

Reply:
fixed